data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -64,21 +65,20 @@ def create_new_dataframe_from_input_CSV_files(
             the input files provided as paramter to this
             function.
         """
+        list_of_file_path_and_names = glob(folder_name+'/*')
         base_file_path_and_name = self.config[KEY_CURRENT_INPUT_FILE]
         for file_path_and_name in list_of_file_path_and_names:
             if not CommonCompHarmQAFunctions.has_same_date_range_in_their_names(
                 base_file_path_and_name,
                 file_path_and_name
             ):
                 raise transform_errors.InputFilesDateRangeMismatchError(base_file_path_and_name, file_path_and_name)
-
```

Comment:
Do not delete empty line! For codes/methods/functions that are long, it is a good thing to put empty lines to show some of these lines are doing something as a unit.